database/tbcd/level/level_test.go

```diff
@@ -0,0 +1,114 @@
+package level_test
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"reflect"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+
+	"github.com/hemilabs/heminetwork/database"
+	"github.com/hemilabs/heminetwork/database/tbcd"
+	"github.com/hemilabs/heminetwork/database/tbcd/level"
+)
+
+func TestMD(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.TODO())
+	defer cancel()
+
+	home := t.TempDir()
+	t.Logf("temp: %v", home)
+
+	cfg := level.NewConfig(home)
+	db, err := level.New(ctx, cfg)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer func() {
+		err := db.Close()
+		if err != nil {
+			t.Fatal(err)
+		}
+	}()
+
+	x := 255
+	rows := make([]tbcd.Row, x+1)
+	for i := 0; i <= x; i++ {
```

Comment:
```suggestion
	for i := range x {
```

Reply:
While I typically agree in this case we need to show the old ugly way because x+1; this is also a test and we want to be explicit and fail if assumptions are wrong.